content/updates/2020-06/1-duplicate-a-product.md

```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Want to easily create a new product based on another? You can now duplicate a simple product in just one click! What a time-saver! :wink:  
+You can find this new action in the `...` button of the product edit form of each one of your products. 
+
+There is a permission to hide this action, if you don't want some users to be able to duplicate products.
```

Comment:
This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.

Reply:
```suggestion
This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.
```